### ISSUE TITLE
0.48 - ionic serve: failed to load resource

### ISSUE THREAD

**Issue by mg1075** [driftyco/ionic-app-scripts#608]
#### Short description of the problem:
After running ionic serve, the result of the build in the browser is: 
`Failed to load resource: the server responded with a status of 404 (Not Found)`
for the file `http://localhost:8100/build/main.js`

#### What behavior are you expecting?
I have an src/app/main.ts file as asked for in the instructions. I am running the ionic conference app. This was working until I updated from 0.47 to 0.48. I think I was updating to 0.48 because android is giving a lengthy white screen at startup (when running on an android device).

**Steps to reproduce:**
1. run `ionic serve`

**Which @ionic/app-scripts version are you using?**
0.48

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

>   ordova CLI: 6.4.0
> Ionic Framework Version: 2.0.0-rc.4
> Ionic CLI Version: 2.1.18
> Ionic App Lib Version: 2.1.9
> Ionic App Scripts Version: 0.0.48
> ios-deploy version: Not installed
> ios-sim version: Not installed
> OS: Windows 7
> Node Version: v6.9.2
> Xcode version: Not installed

**Comment by ivanov84** [comment]
Hi, friend.
I have the same issue with building on windows 7.
I have to copy all files to windws 8 machine and build there.
I'm thinking about moving from win 7 to win 8...

I tryed to install/uninstall nodejs (ver 4.5, 6.9.4) - nothing
Also reinstalled ionic / cordova via npm - nothing.
Added system PATH in Windows - nothing.
Reinstall JDE - nothing.

Result: ionic build system does not build main.js file and main.js.map - although build polyfills.js, 
main.css and sw-toolbox.js - from MyTabs example to be certain.
